packages/flutter/test/widgets/decorated_sliver_test.dart

```diff
@@ -0,0 +1,388 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Can you add tests for reverse in the horizontal and vertical axes?

Reply:
Done!